extensions/sql-database-projects/src/models/project.ts

```diff
@@ -173,14 +175,6 @@ export class Project implements ISqlProject {
 
 		// get publish profiles specified in the sqlproj
 		this._publishProfiles = this.readPublishProfiles();
-
-		// find all SQLCMD variables to include
-		try {
-			this._sqlCmdVariables = utils.readSqlCmdVariables(this.projFileXmlDoc, false);
```

Comment:
I'm guessing this helper function should refactored away at some point.  Still used in the publish dialog, though?

Reply:
yeah it's used for loading sqlcmd variables from the publish profile in the publish dialog. The load publish profile in STS actually also reads the sqlcmd variables from the publish profile, so that should be updated to return those and then we can get rid of the helper function here.